modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -230,22 +230,22 @@ The table below outlines which Cypher version is assigned to databases in differ
 |===
 | Scenario | `db.query.default_language` | Databases
 
-| *Standard DBMS upgrade to Neo4j 2025.05+*
+| *Standard DBMS upgrade to Neo4j 2025.06+*
 | Unset
 | *Existing system database:* `CYPHER 5` +
 *Existing user databases:* `CYPHER 5` +
 *New user databases default:* `CYPHER 5`
 
-| *Custom DBMS upgrade to Neo4j 2025.05+*
-| Manually set to `CYPHER_25` by administrator
+| *Custom DBMS upgrade to Neo4j 2025.06+*
+| Manually set to `CYPHER 25` by administrator
 | *Existing system database:* `CYPHER 5` +
 *Existing user databases:* `CYPHER 5`  +
 *New user databases default:* `CYPHER 25`
 
-| *New installation of Neo4j 2025.05+* 
-| Set to `CYPHER_25`
-| *New system database:* `CYPHER 25` +
-*New user databases default:* `CYPHER 25`
+| *New installation of Neo4j 2025.06+* 
+| Set to `CYPHER 5`
```

Comment:
same here as the comment I unresolved and replied to above. The config value is `CYPHER_5` with an underscore, elsewhere is with space instead. And this column is for the config value.
```suggestion
| Set to `CYPHER_5`
```